L/LZFSE/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LZFSE"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lzfse/lzfse.git", "e634ca58b4821d9f3d560cdc6df5dec02ffc93fd")
```

Comment:
[v1.0](https://github.com/lzfse/lzfse/releases/tag/lzfse-1.0) seems to be https://github.com/lzfse/lzfse/commit/88e2d2788b4021d0b2eb9fe2d97352ae9190f128?

Reply:
Good catch. I updated to the proper commit.